examples/complete/components/terraform/infra/vpc/component.yaml

```diff
@@ -14,6 +14,7 @@ spec:
     # and all URL and archive formats as described in https://github.com/hashicorp/go-getter
     # In 'uri', Golang templates are supported  https://pkg.go.dev/text/template
     # If 'version' is provided, '{{.Version}}' will be replaced with the 'version' value before pulling the files from 'uri'
+    # To vendor a module from a Git repo, use the following format: 'github.com/cloudposse/terraform-aws-ec2-instance.git///?ref={{.Version}}
```

Comment:
Let's not confuse/muddle the purpose of the `component.yaml` in the context of another component.
```suggestion
```